src/python/psic/image_analysis/catalog for image resolution and size.csv

```diff
@@ -0,0 +1,500 @@
+file,size(BYTE),size(KB),size(MB),resolution(DPI)
```

Comment:
Can this file be renamed to something shorter and without spaces in the file name? The code that references it will also have to be updated with the new name!

Reply:
I have updated the file name and also added the new name to the code.